testPkgs/crd/sample_crd_types.go

```diff
@@ -0,0 +1,20 @@
+package crdpkg
```

Comment:
- the package name in the path is `crd`, but the package directive is `crdpkg`. 
- we don't use CRD at all, we use an fybric object directive, so maybe to rename the package to sumthing like fybrikObject

Reply:
Fixed.